evm/check-ownership-request.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const axios = require('axios');
+const { Command, Option } = require('commander');
+const { ethers } = require('hardhat');
+const {
+    getDefaultProvider,
+    Contract,
+    constants: { AddressZero },
+} = ethers;
+
+const { addBaseOptions } = require('./cli-utils');
+const { printError, mainProcessor, printInfo } = require('./utils');
+
+const ERC20 = require('@axelar-network/axelar-cgp-solidity/artifacts/contracts/ERC20.sol/ERC20.json');
+const IInterchainToken = require('@axelar-network/interchain-token-service/interfaces/IInterchainToken.json');
+
+async function processCommand(_, chain, options) {
+    const { rpc } = options;
+    let { address, addresses } = options;
+
+    try {
+        addresses = JSON.parse(addresses);
+    } catch (error) {
+        printError(`Error while parsing addresses: `, error.message);
+    }
+
+    const provider = getDefaultProvider(rpc || chain.rpc);
+    const erc20 = new Contract(address, ERC20.abi, provider);
+
+    try {
+        await checkContract(address, addresses, erc20);
+        await checkAxelarDeployedToken(address, chain.name.toLowerCase(), provider);
+        printInfo(`Contract verification checks passed.`);
+    } catch (error) {
+        printError(`Error while checking contract address`, error.message);
+    }
+}
+
+async function checkContract(address, addressesArray, erc20) {
+    const name = await erc20.name();
+    const symbol = await erc20.symbol();
+    const regex = /axl/i;
```

Comment:
Is this a reliable check? I can imagine how there could be both false-positives or false-negatives:
- If name or token symbol contain `axl` without any intention to mimic Axelar's naming. There are possible words where that can be a substring
- names/symbol of Axelar's tokens aren't guaranteed to contain `axl`.